recipes/vorbis/all/patches/link-libm.patch

```diff
@@ -0,0 +1,13 @@
+--- a/lib/CMakeLists.txt
++++ b/lib/CMakeLists.txt
+@@ -84,6 +84,10 @@ if (NOT BUILD_FRAMEWORK)
+     set_target_properties(vorbisfile PROPERTIES SOVERSION ${VORBISFILE_VERSION_INFO})
+ 
+     target_link_libraries(vorbis ${OGG_LIBRARIES})
++    find_library(M_LIB m)
++    if(M_LIB)
++        target_link_libraries(vorbis ${M_LIB})
++    endif()
```

Comment:
```suggestion
+    if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+        target_link_libraries(vorbis m)
+    endif()
```

Reply:
Ok, but I have to recreate the patch, number of lines is different.